paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -506,6 +631,13 @@ class PaymentSheet internal constructor(
                 this.preferredNetworks = preferredNetworks
             }
 
+            // TODO:(jaynewstrom-stripe) remove before AllowsRemovalOfLastSavedPaymentMethodApi GA.
+            @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+            @ExperimentalAllowsRemovalOfLastSavedPaymentMethodApi
+            fun allowsRemovalOfLastSavedPaymentMethod(allowsRemovalOfLastSavedPaymentMethod: Boolean) = apply {
```

Comment:
This can only be used from the builder! This is the pattern we went with in CustomerSheet, and I think it's the right approach here!